sled-agent/src/params.rs

```diff
@@ -333,15 +333,6 @@ impl OmicronZoneTypeExt for OmicronZoneConfig {
     }
 }
 
-impl crate::smf_helper::Service for OmicronZoneType {
-    fn service_name(&self) -> String {
-        self.kind().service_prefix().to_owned()
-    }
-    fn smf_name(&self) -> String {
-        format!("svc:/oxide/{}", self.service_name())
-    }
-}
-
```

Comment:
Removing this altogether as Omicron zones should not be meddling with smf properties. Only the switch zone still has to unfortunately.